python/dgl/sparse/softmax.py

```diff
@@ -35,8 +35,8 @@ def softmax(A: SparseMatrix) -> SparseMatrix:
     >>> col = torch.tensor([1, 2, 2, 0])
     >>> nnz = len(row)
     >>> val = torch.arange(nnz).float()
-    >>> A = from_coo(row, col, val)
-    >>> softmax(A)
+    >>> A = dglsp.from_coo(row, col, val)
+    >>> dglsp.softmax(A)
```

Comment:
missing `import dgl.sparse as dglsp`

Reply:
Since many other place in Sparse library are assuming the import exists, so I am not adding it in this PR. If we believe we need to import, we can add them in one go.